test/tilejson.test.js

```diff
@@ -419,7 +419,7 @@ tape('findID', function(assert) {
         (setupServer(function() {
             tilejson.getTile(2, 2, 2, function(err, data, headers) {
                 assert.equal(err.code, 'ECONNRESET');
-                assert.equal(connectionCount, 2);
+                assert.equal(connectionCount, 1);
```

Comment:
I think we'll want to leave the `2` here and change the `maxAttempts` here https://github.com/mapbox/node-tilejson/blob/master/lib/tilejson.js#L219 to 2, since it looks like `maxAttempts` setting was a lie until this change in requestretry: https://github.com/FGRibreau/node-request-retry/commit/a130355.